packages/squiggle-lang/__tests__/Distributions/Scale_test.res

```diff
@@ -0,0 +1,19 @@
+/*
+This test case comes via Nuño https://github.com/quantified-uncertainty/squiggle/issues/433
+
+
+*/
+
+open Jest
+open Expect
+open TestHelpers
+
+describe("", () => {
+  test("", () => {
+    let scalelog = DistributionOperation.Constructors.scaleLogarithm(~env, mkExponential(10.0), 2.0)
+
+    E.R2.bind(DistributionOperation.Constructors.mean(~env), scalelog)
+    ->expect
+    ->toEqual(Ok(-2.348336572091017))
```

Comment:
Is this correct?

Reply:
good question-- my source of truth is the code itself, I don't yet have a better way of generating truth.

Reply:
Nuño explained this to me